less/index.less

```diff
@@ -1,12 +1,258 @@
-// Import your less files here:
+// Variables
+// @desktop: 900px;
+// @tablet: 740px;
+// @mobile: 500px;
 
+@desktop: ~'( max-width: 900px )';
+@tablet: ~"( max-width: 740px )";
+@phone: ~"( max-width: 500px )";
 
+// Color pointers
+@nav-bg: @wild-sand;
+@nav-font: @river-bend;
+@nav-border: @sea-shell;
 
+@header-font-color: @ebony-clay;
+@main-font-color: @shuttle-gray;
+@main-border-color: @border-real;
 
+@footer-background: @wild-sand;
+@footer-font-color: @waterloo;
 
+// Color Vars
+@wild-sand: #F6F6F6;
+@river-bend: #454A59;
+@sea-shell: #F1F1F1;
+@ebony-clay: #23293B;
+@shuttle-gray: #5E6164;
+@porcelain: #F6F7F8;
+@waterloo: #81848E;
+@btn-bg-color: #22283A;
+@border-real: #dee2e6;
 
+// Mobile var
+@media @phone {
+    width: 60%;
+    height: 40px;
+    font-size: 2rem;
```

Comment:
Great job with your mobile styles here, but we are missing a few design elements from the design file.